ui/src/config/section/image.js

```diff
@@ -395,6 +395,14 @@ export default {
           show: isZoneCreated,
           component: shallowRef(defineAsyncComponent(() => import('@/views/image/AddKubernetesSupportedVersion.vue')))
         },
+        {
+          api: 'addKubernetesSupportedVersion',
```

Comment:
should this be `getUploadParamsForKubernetesSupportedVersion` instead?

Reply:
Yes. This didn't cause issues, since the component is ignoring the api defined here.